app/controllers/api/v1/departments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class Api::V1::DepartmentsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  before_action :set_department, only: [:show, :edit, :update, :destroy]
+
+  def index
+    @departments = Department.all
+  end
+
+  def create
+    @department = Department.new(department_params)
+    if @department.save
+      render json: { status: :created }
+    else
+      render_errors(@department)
+    end
+  end
+
+  def show
+  end
+
+  def edit    
+  end
+
+  def update
+    if @department.update(department_params)
+      render json: { status: :updated }
+    else
+      render_errors(@department)
+    end    
+  end
+
+  def destroy
+    @department.destroy
+  end
+
+  private
+
+  def set_department
+    @department = Department.find(params[:id])
+  end
+
+  def department_params
+    params.require(:department).permit(:name, :abbreviation)
+  end
+
+  def render_errors(record)
```

Comment:
Since you have `render_errors ` in almost all controllers - it would be good to put it into `application_controller.rb`